components/org.wso2.identity.webhook.wso2.event.handler/src/main/java/org/wso2/identity/webhook/wso2/event/handler/api/builder/WSO2CredentialEventPayloadBuilder.java

```diff
@@ -94,8 +94,8 @@ private PasswordUpdateAction getAction(Flow.Name name) {
         switch (name) {
             case PROFILE_UPDATE:
                 return PasswordUpdateAction.UPDATE;
-            case PASSWORD_RESET:
-                return PasswordUpdateAction.RESET;
+            case CREDENTIAL_RESET:
+                return PasswordUpdateAction.CREDENTIAL_RESET;
```

Comment:
With this change shouldnt we update the event payload to include credential type as well ?